README.md

```diff
@@ -1,8 +1,8 @@
-[![](https://raw.githubusercontent.com/mapbox/mapbox-java/master/.github/splash-img.png)](https://www.mapbox.com/android-docs/mapbox-services/)
```

Comment:
Should we even include a splash image in this repo anymore? I can edit the new one to remove nav if needed but I feel it distracts from the real purpose of the README.md file which is to provide instructions and actual information.